examples/dgmulti_2d/elixir_mhd_weak_blast_wave.jl

```diff
@@ -15,11 +15,9 @@ dg = DGMulti(polydeg=3, element_type = Quad(), approximation_type = Polynomial()
              surface_integral = SurfaceIntegralWeakForm(surface_flux),
              volume_integral = VolumeIntegralFluxDifferencing(volume_flux))
 
-cells_per_dimension = (16, 16)
-vertex_coordinates, EToV = StartUpDG.uniform_mesh(dg.basis.elementType, cells_per_dimension...)
-vertex_coordinates = map(x -> 2 * x, vertex_coordinates) # map domain to [-2, 2]^2
-mesh = VertexMappedMesh(vertex_coordinates, EToV, dg, is_periodic=(true, true))
-
+mesh = DGMultiMesh(dg, cells_per_dimension=(16, 16),
+                     coordinates_min=(-2, -2), coordinates_max=(2, 2),
+                     periodicity=(true, true))
```

Comment:
```suggestion
mesh = DGMultiMesh(dg, cells_per_dimension=(16, 16),
                   coordinates_min=(-2.0, -2.0), coordinates_max=(2.0, 2.0),
                   periodicity=(true, true))
```
This clarifies that we're using `Float64` here.